src/content/docs/codestream/how-use-codestream/pull-requests.mdx

```diff
@@ -1,97 +1,305 @@
+  >
+By default, custom filters use `scope=all` which searches across all of GitLab. To avoid pulling all merge requests or issues listed on GitLab, you will need to refine this search using at least one of the following qualifiers:
+
+* `scope=assigned\_to\_me`
+* `scope=created\_by\_me`
+* `project_id`
+* `group_id`
+* `author_id`
+* `author_username`
+* `assignee_id`
+* `assignee_username`
+* `my\_reaction\_emoji`
+
+Search syntax is derived from GitLab's custom search API. Refer to GitLab's documentation for more details on [merge request searches](https://docs.gitlab.com/ee/api/merge_requests.html) or [issue searches](https://docs.gitlab.com/ee/api/issues.html).
+
+  </Collapser>
+</CollapserGroup>
 
-You can edit a GitHub pull request's details, such as reviewers, assignees and labels.
+## Review a pull request [#review]
 
-![A screenshot showing the details of a PR](/images/PRDetails-GH.png "pull request details")
+<Callout variant="caution">
+Pull requests review is currently not available for Bitbucket.
+</Callout>
 
-For a GitLab merge request, you can use edit mode (via the dropdown at the top of the page) or use the sidebar.
+Regardless of where the pull request was created, you can edit, review, and even merge it from within CodeStream. If you know how to work with pull requests on GitHub or GitLab, you'll know how to do it in CodeStream as well.
 
-![A screenshot showing the details of a merge request](/images/PRDetails-GL.png "merge request details")
+To review a pull request:
 
-By default, you can only add a single reviewer and a single assignee to a GitLab merge request. If your organization supports multiple reviewers and assignees, click the gear menu in the heading of the **Merge Requests** section of the CodeStream pane to enable this.
+1. Check the details:
+* You can edit a GitHub pull request's details, such as reviewers, assignees and labels.
+* For a GitLab merge request, you can use edit mode (via the dropdown at the top of the page) or use the sidebar. By default, you can only add a single reviewer and a single assignee to a GitLab merge request. If your organization supports multiple reviewers and assignees, click the gear menu in the heading of the **Merge Requests** section of the CodeStream pane to enable this.
 
 ![A screenshot showing how to assign multiple users to PR](/images/MultipleSetting-GL.png "multiple settings")
 
-Review the conversation and add comments with the ability to @mention your collaborators. 
+2. Review the conversation and add comments with the ability to @mention your collaborators. 
 
 ![A screenshot showing a conversation in a request](/images/PRConversation-GH.png "pull request conversation")
 
-View the changes, add comments, and submit a review.
+3. View the changes, add comments, and submit a review.
 
-![A screenshot showing a "changed files" view of a PR](/images/PRChanges-GH.png "pull request changes")
+### Perks of reviewing using CodeStream [#review-perks]
```

Comment:
It might be worth considering bringing this section up above the procedural content. Thus:

1. why use CodeStream for PRs?
2. How to use CodeStream for PRs

Just an idea :) I thought this section could go under the intro paragraph, and then the procedural content could get its own header.

Reply:
I like it! I've moved the section up! Thanks!